src/Test.FunctionalTests.Sarif/Partitioning/FilterByPredicateTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
```

Comment:
Sort + Remove usings

Reply:
Fixed.

---
In reply to: [338200662](https://github.com/microsoft/sarif-sdk/pull/1708#discussion_r338200662) [](ancestors = 338200662)